src/components/Avatar/Avatar.tsx

```diff
@@ -12,18 +12,30 @@ const semanticSizes = {
   large: sizeBase * 12,
 };
 
+type TOnErrorArgs = Error | Event | string | null;
+type ImageFetcherArgs = {
+  onLoad: (src: string | null) => void;
+  onError: (e: TOnErrorArgs) => void;
+  src: string | null;
+};
+
 // This function will be used in a useEffect so make sure it returns a cleanup function
-function defaultImageFetcher({ onLoad, onError, src }) {
```

Comment:
I think we can get rid of this implementation and just default to the one provided by the `useImage` hook

Reply:
That sounds great! In the interest of moving forward with the TS conversation let’s make that a new issue and put it on the typescriptification project